src/AtlasModules/atlas-config.bat

```diff
@@ -2065,7 +2066,22 @@ echo Adding extras bucket...
 cmd /c scoop bucket add extras
 goto finish
 
-:browser
+:choco
+echo Installing Chocolatey
+set /P c="Review Install script before executing? [Y/N]: "
+if /I "%c%" EQU "Y" curl "https://community.chocolatey.org/install.ps1" -o %windir%\AtlasModules\install.ps1 && notepad %windir%\AtlasModules\install.ps1
+if /I "%c%" EQU "N" curl "https://community.chocolatey.org/install.ps1" -o %windir%\AtlasModules\install.ps1
+powershell -NoProfile Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('%windir%\AtlasModules\install.ps1'))
```

Comment:
This can probably be reduced to just running the `.ps1` file, the extra code is probably only necessary when downloading Chocolatey.

Reply:
Only did this part to fit with the Scoop install. But I can turn it into one line if you think it's unnecessary to see the script

Reply:
i was only commenting on the powershell part, i think that it can just be reduced to `powershell -NoProfile -EP Unrestricted -Command "%windir%\AtlasModules\install.ps1"`

Reply:
I was just doing what I was advised by the chocolatey webpage but will try this method and get back to this

Reply:
I've done that and it works, so pushed a commit! Hopefully you can review and give the thumbs up!